.github/CONTRIBUTING.md

```diff
@@ -59,11 +59,11 @@ $ glide install --strip-vendor
 # generate (Only required to integrate other components such as web dashboard)
 $ go generate
 # Standard go build
-$ go build
-# Using gox to build multiple platform
-$ gox "linux darwin" "386 amd64 arm" \
-    -output="dist/traefik_{{.OS}}-{{.Arch}}"
+$ go build -o dist/traefik ./cmd/
```

Comment:
Do you really want to specify the output path? the binary is in the gitignore for the root directory. Instead, could you add a shim build in root that just "forwards" to `./cmd`? Sorry for the terrible terminology, but go build is such a simple dev command, that we really should fight to keep ;)